src/Diagnostics.ModelsAndUtils/Utilities/Constants.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Diagnostics.ModelsAndUtils.Utilities
+{
+    public static class SolutionConstants
+    {
+        public static readonly string AppRestartDescription = @"
+ #### App Restart
```

Comment:
Do you think its better to have this content somewhere outside the backend project because any small change in this would need a full backend deployment in production. 

I can think of following options:
1) Having it be stored as Github doc and backend reads and caches the doc (based on etag changes). This is a longer route however will give you most flexibility for future changes.
2) Having them in the UI project : Its a shorter route right now and to change the description in future, it would need a UI deployment which is much cheaper to do.

Reply:
Good point, I'll move them to the frontend for now.